core/issuers.js

```diff
@@ -123,14 +176,16 @@ Description must be truthy: ${description}`;
       // for non-fungible tokens that are unique, destroy them by removing them from
       // the purses/payments that they live in
       mintController.destroy(amount);
+      if (childMint !== undefined) {
+        childMint.destroy(getChildAmount(amount)); // recursively destroy child assets
+      }
     },
     revoke(amount) {
-      this.destroy(amount);
-      return mint(amount);
+      mint.destroy(amount);
+      return mint.mint(amount);
     },
-    mint(initialBalance, _name = 'a purse') {
+    mint(initialBalance, _purseName = 'a purse') {
```

Comment:
Why are we still coercing `_name` on line 52 above, but no longer coercing `_purseName` here?

Reply:
This purseName never gets used. I'm not sure how that happened, but it happened before this PR.

Reply:
I created a new PR that adds getName() methods to actually be able to use `name` https://github.com/Agoric/ERTP/pull/41/